src/main/java/org/opensearch/eval/Constants.java

```diff
@@ -0,0 +1,75 @@
+package org.opensearch.eval;
+
+public class Constants {
+
+    /**
+     * The name of the index that stores the implicit judgments.
+     */
+    public static final String JUDGMENTS_INDEX_NAME = "judgments";
+
+    /**
+     * The name of the UBI index containing the queries. This should not be changed.
+     */
+    public static final String UBI_QUERIES_INDEX_NAME = "ubi_queries";
+
+    /**
+     * The name of the UBI index containing the events. This should not be changed.
+     */
+    public static final String UBI_EVENTS_INDEX_NAME = "ubi_events";
+
+    /**
+     * The name of the index that stores the query sets.
+     */
+    public final static String QUERY_SETS_INDEX_NAME = "search_quality_eval_query_sets";
+
+    /**
+     * The name of the index that stores the metrics for the dashboard.
+     */
+    public final static String DASHBOARD_METRICS_INDEX_NAME = "sqe_metrics_sample_data";
```

Comment:
We changed that to `sqe_metrics` in https://github.com/o19s/opensearch-search-quality-evaluation/pull/91